### PR TITLE
predict with model 

### PR DESCRIPTION
**Pull request by gtmeier**
make predicted water masks given vh vv sar images.
wrapper function to predict masks in batches.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -66,6 +79,132 @@ def test_wrapper(args: Namespace) -> None:
         plot_predictions(details['Percent'], args.dataset)
 
 
+def make_predict_metadata(tile_path: str) -> List[Tuple[str, str]]:
+    predict_metadata = []
+    vh_vv_list = os.listdir(tile_path)
+    vh_vv_list = natsorted(vh_vv_list)
```
Comment (mckadesorensen) [inline main.py]: vh_vv_list = natsorted(os.listdir(tile_path)) might look a little cleaner

File: main.py [hunk 2]
```diff
@@ -66,6 +79,132 @@ def test_wrapper(args: Namespace) -> None:
...
+    midpoint = int(len(vh_vv_list)/2)
+    vh_list = vh_vv_list[0:midpoint]
+    for i in range(midpoint):
+        vh_list[i] = os.path.join(tile_path, vh_list[i])
+    vv_list = vh_vv_list[midpoint:]
+    for i in range(midpoint):
```
Comment (mckadesorensen) [inline main.py]: Combining these for loops might look a little cleaner if it's possible.

File: main.py [hunk 3]
```diff
@@ -66,6 +79,132 @@ def test_wrapper(args: Namespace) -> None:
...
+    for sar in os.listdir(input_sar_path):
+        if not os.path.isdir(os.path.join(input_sar_path, sar)):
+            tif = os.path.join(input_sar_path, sar)
+            tifData = gdal.Open(tif)
+            xStep, yStep = mxm_tile_size, mxm_tile_size
+            xSize, ySize = tifData.RasterXSize, tifData.RasterYSize
+            count = 0
+            for x in range(0, xSize, xStep):
+                for y in range(0, ySize, yStep):
+                    # fileName = '<Image|Mask>_<sar>_<0-9+>.tif'
```
Comment (mckadesorensen) [inline main.py]: If this like isn't needed we might want to delete it instead of commenting it out.

File: model_finder.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+def getFileName():
+    root = Tk()
+    filename = askopenfilename()
+    root.destroy()
+    return filename
+
+
+def main():
+    # get inputs
+    model_path = 'models/ai_test_5'
```
Comment (mckadesorensen) [inline model_finder.py]: We might not want to hard code in ai_test_5

Comment (gtmeier) [reply model_finder.py]: removing

**Comment by gtmeier** [comment]
duplicate code. we now use rohan's

**Pull request closed without merging**